site/rcuinfo.md

```diff
@@ -0,0 +1,45 @@
+## Specifying RCU connection information in the model
+
+During creating or updating a domain, there is new section RCUDbInfo in the model, under the domainInfo section, for specifying RCU database connection information without using the command-line options `-rcu_database` and `-rcu_prefix`. Use this to support a database where the connection string is more complex and requires extra options.
```

Comment:
new section RCUDbInfo in the model, under the domainInfo section -> new section `RCUDbInfo` in the model, under the `domainInfo` section (use a code font for code words)